isolates/bin/send_and_receive.dart

```diff
@@ -0,0 +1,44 @@
+// Read the file, spawn an isolate, send the file contents to the spawned
+// isolate, and wait for the parsed JSON.
+import 'dart:async';
+import 'dart:convert';
+import 'dart:io';
+import 'dart:isolate';
+
+final filename = 'json_01.json';
+
+Future<void> main() async {
```

Comment:
Might be nice to have @lrhn or @mkustermann review this!

Reply:
Resolving this as Lasse reviewed it